app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/LearningSettingsActivity.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.HumanLearning2021.HumanLearningApp.view
+
+
+import android.content.Intent
+import android.os.Build
+import android.os.Bundle
+
+import android.widget.Button
+import androidx.annotation.RequiresApi
+import androidx.appcompat.app.AppCompatActivity
+import com.github.HumanLearning2021.HumanLearningApp.R
+
+const val EXTRA_LEARNING_MODE = "com.github.HumanLearning2021.HumanLearningApp.view.EXTRA_LEARNING_MODE"
+
+class LearningSettingsActivity : AppCompatActivity() {
+
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_learning_settings)
+
+        val intent = Intent(this, LearningActivity::class.java)
+
+        val btPres = findViewById<Button>(R.id.learningSettings_btChoosePresentation)
+        val btRep = findViewById<Button>(R.id.learningSettings_btChooseRepresentation)
+
+        btPres.setOnClickListener {
+            intent.putExtra(EXTRA_LEARNING_MODE, LearningMode.PRESENTATION)
+            startActivity(intent)
+        }
+        btPres.tooltipText = "@string/learning_settings_tooltip_presentation"
```

Comment:
that's not how this works x) see : https://developer.android.com/reference/android/content/res/Resources#getString(int)

I'll fix it for you